python_modules/libraries/dagster-dask/dagster_dask/data_frame.py

```diff
@@ -288,8 +287,7 @@
         "options": {
             "path": (Any, True, "Path to a target filename."),
             "key": (String, True, "Datapath within the files."),
-            "compute": (Bool, False, "Whether or not to execute immediately."),
```

Comment:
did you mean to remove this? just concerned about breaking changes

Reply:
Yes, all the `compute` options have been removed, because they don't make sense in the context of materialization configs. Setting `compute=False` will cause Dask to return a future, which in the materialization flow will never be computed. This would result in `AssetMaterialization` objects being yielded for assets that don't actually exist.

Reply:
Can you add a note to `Changes.md` at the repo root - something like
```
## 0.9.6 (Upcoming)
**Breaking Changes**
* [dagster-dask] removed the `compute` options key which would result in un-executed futures if used 
```

but add some more context on where that key is

Reply:
Added a note to the changelog.

Reply:
Rebased and updated for post-0.9.6.